src/vt-tv/api/info.h

```diff
@@ -81,9 +79,8 @@ struct Info {
    * \param[in] object_info object information to merge with existing data
    * \param[in] r the rank work
    */
-  void addInfo(
-    std::unordered_map<ElementIDType, ObjectInfo> object_info, Rank r
-  ) {
+  void
+  addInfo(std::unordered_map<ElementIDType, ObjectInfo> object_info, Rank r) {
```

Comment:
This was probably okay how it was--was this change from running .clang-format or is this specified in the vt style guidelines?